crowbar_framework/app/models/service_object.rb

```diff
@@ -1551,26 +1544,6 @@ def run_remote_chef_client(node, command, logfile_name)
     }
   end
 
-  def chef_daemon(action, node_list)
```

Comment:
... just for rference, i mean https://github.com/crowbar/barclamp-crowbar/commit/75fbade214b9c85d7a1cb78831ddffc419a51454 and https://github.com/crowbar/barclamp-crowbar/commit/a4b4397595be5dfd0383b278a79da46dbb3b946a